src/pages/api/user/[id]/experience/index.js

```diff
@@ -23,18 +24,33 @@ export default async function handler(req, res) {
       // Create an experience for an user and post it to the database
       case 'POST':
         const experience = new ExperienceModel({
```

Comment:
Is this going to include the two submodules that we talked about? See [Backend Meeting](https://docs.google.com/document/d/1ptGG2C2oB9NXGKJC0VPB-p_FbxdNwkj6VrEdm0lVwok/edit?usp=sharing) Notes from Nov. 4.

Reply:
Not for this milestone. For the showcase module I would have to implement media storage management (images, videos, docs, etc.), which I didn't have time to do for this milestone.

Reply:
Understood. Thanks!